examples/CMakeLists.txt

```diff
@@ -1,40 +1,55 @@
-add_executable(benchmarks_all benchmarks/main.cpp)
+
+file(GLOB_RECURSE tiny_dnn_headers "${CMAKE_SOURCE_DIR}/tiny_dnn/*.h")
```

Comment:
@beru this is not needed since during the cmake configuration we include the sources in the target
https://github.com/tiny-dnn/tiny-dnn/blob/master/CMakeLists.txt#L299